Amazon.IonHashDotnet/Serializer.cs

```diff
@@ -299,9 +307,9 @@ private int GetLengthLength(byte[] bytes)
             {
                 // symbols are serialized as strings; use the correct TQ:
                 tq = 0x70;
-                if (isNull)
+                if (ionValue.Text == null && ionValue.Sid == 0)
                 {
-                    tq |= 0x0F;
+                    tq = 0x71;
                 }
```

Comment:
doesn't this change break the `null.symbol` case?

Reply:
`null.symbol` returns a zero symbol

```
            IIonReader reader = IonReaderBuilder.Build("null.symbol");
            reader.MoveNext();
            SymbolToken symbolToken = reader.SymbolValue();  //text=null, sid=0
```
tq will be set to 0x71 to handle zero symbol

Reply:
i see, thank you; yes, i agree passing `isNull` adds clarity